doc/sphinx/source/howto/distribute-train.rst

```diff
@@ -1,17 +1,28 @@
 .. howto/distribute-train.rst 
 
 
-Train using multiple nGraph CPU backends with data parallel 
-===========================================================
+Distribute training across multiple nGraph backends 
+===================================================
 
 In the :doc:`previous section <../howto/derive-for-training>`, we described the 
 steps needed to create a "trainable" nGraph model. Here we demonstrate how to 
 train a data parallel model by distributing the graph to more than one device.
 
-As of release version 0.12, the default build is with OpenMPI.  To use the 
-`Intel MLSL`_ library, set the following compilation flag at build time: 
+These options are currently supported for available backends; choose the best one 
+for your scenario.  
 
-``-DNGRAPH_DISTRIBUTED_ENABLE=TRUE``.
+* Use ``-DNGRAPH_DISTRIBUTED_OMPI_ENABLE=TRUE`` to enable distributed training with 
+  OpenMPI. Use of this flag requires that OpenMPI be a pre-existing library 
+  in the system. If it's not present on the system, install Open MPI 2.1.1 before 
```

Comment:
Why 2.1.1. If it's a default version in Ubuntu18, we might want to say some thing like, at least 1.10 (default in Ubuntu16)

Reply:
Our CI is based on 2.1.1 
we can probably say 2.1.1 or greater ?

Reply:
This is what Sandeep said it should be... let me know if it needs fixed